redis.conf

```diff
@@ -0,0 +1,116 @@
+#################################### SETUP ####################################
+
+./redis-server /acapy-plugin-pickup/redis.conf
+
+protected-mode yes
+port 6379
+
+timeout 0
+
+tcp-keepalive 300
+
+################################### GENERAL ###################################
+
+daemonize-no
```

Comment:
should it be `daemonize no` instead of `daemonize-no`?

Reply:
You are right!